util/tippingContractUtil.js

```diff
@@ -63,14 +120,16 @@ tippingContractUtil.getTipsRetips = (state) => {
     }
 
     data.type = data.type ? data.type : 'AeTip';
-    data.id = id;
+    data.id = id + suffix;
+    data.contractId = returnState.result.contractId;
 
-    const hasClaim = data.claim_gen !== undefined && data.url_id !== undefined;
-    data.url = hasClaim ? findUrl(data.url_id) : null;
-    data.retips = hasClaim ? findRetips(id, data.url_id) : [];
-    data.claim = hasClaim ? findClaimGen(data.claim_gen, data.url_id) : null;
+    data.url = data.url_id !== undefined ? findUrl(data.url_id) : null;
+    data.retips = data.url_id !== undefined ? findRetips(id, data.url_id) : [];
 
-    data.token = data.token ? data.token : null;
+    data.claim_gen = data.claim_gen === "None" || data.claim_gen === undefined ? null : data.claim_gen;
```

Comment:
whats the goal of this expression in case claim_gen === "None"? Right now it results in `null` although I would expect it to return `true`. Could you elaborate a bit?

Reply:
maybe this can be removed now, but during development I had issues with the `option` datatype returned from the contract which was in some cases `"None"` or `value`.

Reply:
doesn't seem needed any more, removed the `None` check, I still want to convert undefined to null for later handling